config/webhook/manifests.yaml

```diff
@@ -1,13 +1,14 @@
 
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   creationTimestamp: null
   name: mutating-webhook-configuration
 webhooks:
-- clientConfig:
-    caBundle: Cg==
```

Comment:
why is this removed?

Reply:
> Mostly looks good to me, however text became lengthy(I guess auto edited) and hard to read (in github UI) in the manifests.
> 
> One question inline:

Text is auto generated. Nothing to do there. Checking why cabundle issue

Reply:
@furkatgofurov7 I believe this is a by product of using the newest controller-tool v0.5.0. Check here https://github.com/kubernetes-sigs/cluster-api/commit/e9cacc0931eff0118442a5e596b64eccf380c731#diff-369b61dd1f2f1f60722927fda70e5a51e130cea68336e6d536086b522ffef0c6L56

Reply:
Yes, bumped into that at some point earlier, was not sure if it was removed automatically or not, thanks for pointing out!